src/Compilers/Core/Portable/DocumentationCommentId.cs

```diff
@@ -301,22 +314,39 @@ private static string DecodePropertyName(string name, string language)
             return name;
         }
 
-        private class DeclarationGenerator : SymbolVisitor
+        /// <summary>
+        /// The <see langword="bool"/> generic type argument indicates if we succeeded at writing out the declaration ID
+        /// or not.  Callers should only call into <see cref="SymbolVisitor{TResult}.Visit(ISymbol?)"/> and should check
```

Comment:
It looks like this statement is product of earlier iterations and no longer relevant.

Reply:
indeed!  will fix.